lacci/lib/shoes/drawables/font_helper.rb

```diff
@@ -0,0 +1,62 @@
+module FontHelper
+
+    def parse_font(font)
+        
+        input = font
+        regex = /\s+(?=(?:[^']*'[^']*')*[^']*$)(?![^']*,[^']*')/
+        result = input.split(regex)
+       
+        fs = nil
```

Comment:
This is an excellent contribution. Thank you!

One thought, no need to take action but thinking out loud.

For readability in future it could be nice to use descriptive variable names; for what concept they are representing. 

(Thinking about the "fs, foss, etc.")

Thanks again!

Reply:
Thanks for the advice , will keep that in mind while writing code in future : )